pkg/query/logical/measure/measure_plan_aggregation.go

```diff
@@ -39,18 +39,26 @@ var (
 )
 
 type unresolvedAggregation struct {
-	unresolvedInput  logical.UnresolvedPlan
-	aggregationField *logical.Field
-	aggrFunc         modelv1.AggregationFunction
-	isGroup          bool
+	unresolvedInput         logical.UnresolvedPlan
+	aggregationField        *logical.Field
+	aggrFunc                modelv1.AggregationFunction
+	isGroup                 bool
+	needCompletePushDownAgg bool
 }
 
-func newUnresolvedAggregation(input logical.UnresolvedPlan, aggrField *logical.Field, aggrFunc modelv1.AggregationFunction, isGroup bool) logical.UnresolvedPlan {
+func newUnresolvedAggregation(
+	input logical.UnresolvedPlan,
+	aggrField *logical.Field,
+	aggrFunc modelv1.AggregationFunction,
+	isGroup bool,
+	needCompletePushDownAgg bool,
```

Comment:
```suggestion
```

To aggregate a node and access the first field, modify the schema returned by distributedPlan.Schema(). The new schema.CreateFieldRef() should produce a field that represents the aggregated result.